docs/Integrating-with-Finch/Integrate-Finch-Connect/Embed-Connect.md

```diff
@@ -25,9 +24,9 @@ Parameter | Required | Description
 ---------|----------|---------
  `clientId` | true | Your `client_id`, a unique identifier for your application.
  `products` | true | An array of permissions your application is requesting access to. See [here](../../Development-Guides/Permissions.md) for a list of valid permissions.
- `payrollProvider` | false | An optional parameter that allows you to bypass the employment system selection screen by providing a valid Provider `id`. Read [here](../../Development-Guides/Providers.md) for more information.
- `sandbox` | false | An optional value that allows users to switch on the sandbox mode to login with fake credentials and test applications against mock data. Read more on testing [here](../../Development-Guides/Testing.md).
- `manual` | false | An optional value which when set to true displays both automated and Assisted Connect employment systems on the selection screen. Read more about Assisted Connect here.
+ `payrollProvider` | false | An optional parameter that allows you to bypass the providers selection screen by providing a valid provider `id`. Read [here](../../Development-Guides/Providers.md) for more information.
```

Comment:
I just caught this during my re-review, I think this `provider selection screen` is grammatically more correct. I think.

Reply:
@sshariff01 Can you also look at `/Integrating-With-Finch/Integrate-Finch-Connect/Redirect-to-Connect.md`? It seems that one has `payrollProvider` and the other `payroll_provider`. Which one is correct?

Reply:
Just took a look. They are both correct in their respective contexts. `payrollProvider` is what's used with the React SDK. `payroll_provider` is what's used when constructing the authorization URL directly.

Reply:
Okay, good to know. Thanks.